src/main/java/com/cmsc495/ticketsystem/controller/AdminController.java

```diff
@@ -24,11 +29,47 @@ public class AdminController {
     @Autowired
     private TicketService ticketService;
 
+    @Autowired
+    private MyUserDetailService myUserDetailService;
 
     @GetMapping("/admin")
     public String showAdminPage(Model model) {
         List<TicketModel> ticketModelList = ticketService.findAllTickets();
         model.addAttribute("tickets", ticketModelList);
         return "admin";
     }
-}
+
```

Comment:
All of this looks great but it should be on a ManageUsersController like how we have a new LoginController.

See TicketController's relationship with AdminController as an example

Reply:
Created class ManageUsersController and moved logic from AdminController related to users there